config-xml/rule-schema.xsd

```diff
@@ -264,7 +265,7 @@
 							</xs:element>
 							<xs:element name="sourceTechnology" type="technology" maxOccurs="unbounded" minOccurs="0"/>
 							<xs:element name="targetTechnology" type="technology" maxOccurs="unbounded" minOccurs="0"/>
-							<xs:element name="tags">
+							<xs:element name="tags" maxOccurs="1" minOccurs="0">
```

Comment:
tags is not used now. Feel free to yield it to me, I have changed it in another PR already